cylc/flow/scheduler.py

```diff
@@ -253,6 +253,7 @@ def __init__(self, reg, options):
         self._profile_amounts = {}
         self._profile_update_times = {}
         self.pre_submit_tasks = []
+        self.badhosts: Set[str] = set()
```

Comment:
```suggestion
        self.bad_hosts: Set[str] = set()
```
(we don't usually normally ram two words together like this)